gammapy/estimators/points/core.py

```diff
@@ -139,14 +142,20 @@ def read(
             kwargs.setdefault("format", "ascii.ecsv")
             table = Table.read(filename, **kwargs)
 
+        try:
+            gti = GTI.read(filename)
```

Comment:
Also you could rather open the `HDUList` and then read the `Table` and `GTI` at their correct locations. 
Of course one has to be careful maintaining backward compatibility with files having no extname defined.